### ISSUE TITLE
Visual Studio MAUI Comet Template renders blank screen on Windows Machine

### ISSUE THREAD

**Issue by TitusRie** [dotnet/Comet#238]
It looks like the latest Comet Template (included with Visual Studio 2022 Preview - Version 17.1.0 Preview 3.0) doesn't render on a Windows Machine, while the Android version looks fine.

I'm not sure this is the right way and right place to open this issue, please let me know if there's a better way or place :-)

Steps to reproduce:

1. Create new .NET MAUI Comet App project:
![image](https://user-images.githubusercontent.com/69340144/151159017-48b3f580-4c66-447d-b548-af7b2baf4d13.png)

2. Use default values for versions:
![image](https://user-images.githubusercontent.com/69340144/151159107-73604d99-ea13-4687-895c-29e51dc5d221.png)

3. Fix missing using:
![image](https://user-images.githubusercontent.com/69340144/151159300-1e2b65d4-9a02-49c5-b94f-f632313cc526.png)

4. Run Debug configuration on Windows Machine:
![image](https://user-images.githubusercontent.com/69340144/151159398-8631faa2-ca38-4bc8-905e-b0c7076f558b.png)

**Comment by Clancey** [comment]
Yes, Comet will not work properly  on windows until Preview 13. There are few commits to Maui.Core that are needed for it to worl.

**Comment by TitusRie** [comment]
Thanks, good to know it should be fixed in the coming release(s) of MAUI :-)
Shall I keep this issue open and check with the coming release if it works?

For reference the links:
[MAUI Blog](https://devblogs.microsoft.com/dotnet/category/maui/)
[MAUI Roadmap](https://github.com/dotnet/maui/wiki/roadmap)

**Comment by sasaju** [comment]
Is now the problem solved? I meet this error.


https://user-images.githubusercontent.com/21136684/178111640-2324e2bb-7c54-4235-85e5-4c245e19902b.mp4
